main/busybox/busyboxconfig-extras

```diff
@@ -376,6 +372,14 @@ CONFIG_DEFAULT_SETFONT_DIR=""
 # CONFIG_FEATURE_START_STOP_DAEMON_FANCY is not set
 # CONFIG_WHICH is not set
 
+#
+# klibc-utils
```

Comment:
I invoked `make menuconfig` for both config files to update them for the new release. Busybox has introduced these 3 new applets in 1.28 which were enabled by `make menuconfig`, if we don't want these we should disable them.

Reply:
Decided against enabling these by default.